vagrant/install-on-travis-ci.sh

```diff
@@ -19,7 +20,7 @@ sudo apt-get update -qq
 sudo apt-get install -y -qq libboost-dev libboost-system-dev \
                         libboost-filesystem-dev libexpat1-dev zlib1g-dev libxml2-dev\
                         libbz2-dev libpq-dev libgeos-c1 libgeos++-dev libproj-dev \
-                        postgresql-server-dev-9.3 postgresql-9.3-postgis-2.1 postgresql-contrib-9.3 \
+                        postgresql-server-dev-9.5 \
```

Comment:
It's a bit hidden on https://docs.travis-ci.com/user/database-setup/#PostgreSQL "All installed versions of PostgreSQL include PostGIS 2.1"